homeassistant/components/onvif/camera.py

```diff
@@ -194,33 +194,36 @@ def __init__(self, hass, config):
             system_date = dt_util.utcnow()
             device_time = await devicemgmt.GetSystemDateAndTime()
             if device_time:
-                cdate = device_time.UTCDateTime
-                cam_date = dt.datetime(
-                    cdate.Date.Year,
-                    cdate.Date.Month,
-                    cdate.Date.Day,
-                    cdate.Time.Hour,
-                    cdate.Time.Minute,
-                    cdate.Time.Second,
-                    0,
-                    dt_util.UTC,
-                )
+                cdate = device_time.UTCDateTime or device_time.LocalDateTime
+                if cdate is None:
+                    _LOGGER.warning("Could not retrieve date/time on this camera")
+                else:
+                    cam_date = dt.datetime(
+                        cdate.Date.Year,
+                        cdate.Date.Month,
+                        cdate.Date.Day,
+                        cdate.Time.Hour,
+                        cdate.Time.Minute,
+                        cdate.Time.Second,
+                        0,
+                        dt_util.UTC,
```

Comment:
`LocalDateTime` is not `UTC`. Would be safer to assume it's local timezone (👍) and so use `dt_util.DEFAULT_TIME_ZONE` in that case ?